crates/pop-contracts/src/call.rs

```diff
@@ -12,9 +12,12 @@ use subxt::{Config, PolkadotConfig as DefaultConfig};
 use subxt_signer::sr25519::Keypair;
 use url::Url;
 
```

Comment:
Unnecessary newline. Please enforce alphabetical order and no newlines between import of crates (locally or external)